assert.js

```diff
@@ -37,6 +37,8 @@ const { isPromise, isRegExp } = require('util/').types;
 const objectAssign = Object.assign ? Object.assign : require('es6-object-assign').assign;
 const objectIs = Object.is ? Object.is : require('object-is');
 
+const RegExpPrototypeTest = require('call-bind/callBound')('RegExp.prototype.test');
```

Comment:
call-bind needs to be added to package.json also

Reply:
@ljharb Thanks, added